### PR TITLE
fix: generativeaionvertexai_gemini_get_started

### PR DESCRIPTION
**Pull request by TetyanaYahodska**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist

- [x] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [ ] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] **Tests** pass:   `mvn clean verify` **required**
- [x] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample 
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by TetyanaYahodska** [comment]
Changed img for Quickstart.java for language consistency.

**Comment by Sita04** [comment]
@TetyanaYahodska Should the corresponding test be updated to account for the image change?

**Review by Sita04 — APPROVED** [review]
Please address the open comment before merging.

File: vertexai/snippets/src/test/java/vertexai/gemini/SnippetsIT.java [hunk 1]
```diff
@@ -172,9 +172,8 @@ public void testSimpleQuestionAnswer() throws Exception {
 
   @Test
   public void testQuickstart() throws IOException {
-    String output = Quickstart.quickstart(PROJECT_ID, LOCATION, GEMINI_FLASH);
-    // Disabled assertion, pending resolution of b/342637034
-    // assertThat(output).contains("Colosseum");
+    Quickstart.quickstart(PROJECT_ID, LOCATION, GEMINI_FLASH);
+    assertThat(bout.toString()).contains("scones");
```
[the remark this answers is not in the document]
Comment (TetyanaYahodska) [reply vertexai/snippets/src/test/java/vertexai/gemini/SnippetsIT.java]: I don't use sysout and use assert from the returned string of this method. Could you elaborate on your point?

[the remark this answers is not in the document]
Comment (TetyanaYahodska) [reply vertexai/snippets/src/test/java/vertexai/gemini/SnippetsIT.java]: Could you review the last commit, please? This code is outdated.

**Pull request merged**